src/main/kotlin/com/motycka/edu/lesson02/Functions.kt

```
@@ -11,20 +12,60 @@ fun main() {
 
 /* Implement the functions below */
 
+
 fun processOrder(items: List<String>, payment: Double): Double {
-    val orderId = TODO("call placerOrder(items)")
-    val totalToPay = TODO("call payOrder(orderId)")
+    val orderId = placerOrder(items) // call placerOrder(items)
+    val totalToPay = payOrder(orderId) // call payOrder(orderId)
+
+    val change = payment - totalToPay // calculate change by subtracting totalToPay from payment
 
-    val change = TODO("calculate change by subtracting totalToPay from payment")
+    if (change < 0) {
+        println("Order $orderId failed: Insufficient payment.")
+        return -1.0
```

would be better to throw an error. What would happen with return -1 if there was a real customer?